src/base/io/utilities/convertOldStyleModel.m

```diff
@@ -127,15 +127,19 @@
     end
 end
 
-if ~isfield(model,'osense')
-    if isfield(model,'osenseStr')
-        if strcmpi(model.osenseStr,'min')
-            model.osense = 1;
-        else
-            model.osense = -1;
+if isfield(model,'osense')
```

Comment:
See my comment above for a separate function.

Reply:
Not possible here. as this is osense -> osenseStr conversion (osense is no longer a model field, so it should not be considered in any function for models).